src/main/java/cr0s/warpdrive/config/Dictionary.java

```diff
@@ -816,6 +818,8 @@ public static boolean isLeftBehind(final Entity entity) {
 	}
 	
 	public static boolean isLivingWithoutAir(final Entity entity) {
+		if (WarpDriveConfig.isMatterOverdriveLoaded && entity instanceof EntityPlayer) return CompatMatterOverdrive.isAndroid((EntityPlayer) entity);
```

Comment:
one condition statement per line means that:
if ( WarpDriveConfig.isMatterOverdriveLoaded
 && entity instanceof EntityPlayer ) {
 return CompatMatterOverdrive.isAndroid((EntityPlayer) entity);
}

This helps readability of the conditions and improves versionning

Reply:
so my recent commit was ambiguous and need to be reverted?

Reply:
ok.